website/docs/Installation/macos-install.md

```diff
@@ -5,8 +5,6 @@ sidebar_position: 3
 # MacOS
 ## Installing Podman Desktop on MacOS using .dmg file
 
-#### Prerequisite: Podman Machine
-
 If you already have a Podman machine up and running, you can check out the [Downloads](/downloads/macos) section of this website to get the applicable .dmg file depending on your Mac Hardware Architecture (i.e. Intel or Apple M1).
```

Comment:
That's the first thing the reader will read.
If I'm new to podman, I'd have no idea what a "podman machine" is.

Reply:
Then, there are multiple binaries, the document should explain to the reader how to chose the right one. The universal binary being here to simplify things, so the user does not have to worry/know about which architecture they have (--> should be the default/recommended path).